docusaurus/docs/reference/config-types/host.v2.md

```diff
@@ -0,0 +1,291 @@
+---
+sidebar_label: host.v2
+sidebar_position: 20
+---
+
+# The `Host.v2` Config Type
+
+The `host.v2` configuration type defines how edge routers or tunnelers should create terminators and
+health checks for the application endpoints that provide a service.
+
+## Terminators
+
+Each config can define one or more terminators. If a service is hosted by multiple servers, the
+configuration can have a terminator defined for each hosting server.
+
+For services with defined endpoints, the following three attributes are required:
+
+* `address` - an IP or DNS address to connect to
+* `port` - the network port to connect to
+* `protocol` - tcp or udp
+
+**Examples**
+
+This service is hosted by one application server.
+
+```json
+{
+  "terminators": [
+    {
+      "address": "192.168.100.1",
+      "port": 80,
+      "protocol": "tcp"
+    }
+  ]
+}
+```
+
+This service is hosted by two application servers.
+
+```json
+{
+  "terminators": [
+    {
+      "address": "192.168.100.1",
+      "port": 80,
+      "protocol": "tcp"
+    },
+    {
+      "address": "192.168.100.2",
+      "port": 80,
+      "protocol": "tcp"
+    }
+  ]
+}
+```
+
+Services that are forwarding traffic from an OpenZiti tunneling proxy may use the following
+properties to indicate what should be forwarded:
+
+* `forwardProtocol` - flag indicating that the protocol of the forwarded connection is to be used
+    * Must be the value `true`
+* `allowedProtocols` - the list of allowed protocols. Valid values include `tcp` and `udp`
+* `forwardAddress` - flag indicating that the target address of the forwarded connection is to be
+  used
+    * Must be the value `true`
+* `allowedAddresses` - the list of allowed addresses. Valid values include ips, hostnames and CIDRs
+* `forwardPort` - flag indicating that the target port of the forwarded connection is to be used
+    * Must be the value `true`
+* `allowedPortRanges` - the list of allowed port ranges.
+    * Example: `allowedPortRanges: [ {"low" : 80, "high" : 80}, {"low" : 8080, "high" : 8090} ]`
+* `allowedSourceAddresses` - list of addresses in ip, hostname or CIDR
+    * hosting tunnelers establish local routes for the specified source addresses so binding will
+      succeed
+
+**Examples**
+
+```json
+{
+  "terminators": [
+    {
+      "forwardAddress": true,
+      "allowedAddresses": [
+        "192.168.1.0/24",
+        "10.0.0.1/16"
+      ],
+      "forwardPort": true,
+      "allowedPorts": [
+        {
+          "low": "1024",
+          "high": 2048
+        }
+      ],
+      "forwardProtocol": true,
+      "allowedProtocols": [
+        "tcp",
+        "udp"
+      ]
+    }
+  ]
+}
+```
+
+Note that not everything must be forwarded. For example the address could be hard coded, while the
```

Comment:
> not everything must be forwarded

Maybe, something like: "Note that the address is not forwarded in the example below. The port and protocol are forwarded and the corresponding 'allow' is set, but the address is statically set to '192.168.100.1" ?

Reply:
fixed